packages/ui/useScroll/useScroll.js

```diff
@@ -728,6 +728,10 @@ const useScrollBase = (props) => {
 			setScrollTop(scrollTop);
 		}
 
+		if (scrollMode === 'native') {
+			scrollContentHandle.current.updateScrollPositionTarget(mutableRef.current.scrollLeft, mutableRef.current.scrollTop);
+		}
+
 		if (scrollContentHandle.current.didScroll) {
```

Comment:
Why don't we set `scrollPositionTarget` in `didScroll` function?
It seems either `native` or `translate` mode are all calling `didScroll` after scroll.

Reply:
Maybe we can remove updateScrollPositionTarget function and the same code block from `setScrollPosition` and `scrollToPosition`.

Reply:
So, in `didScroll`,
```
this.scrollPosition = pos;
this.scrollPositionTarget = pos;
```
will do the same thing in updateScrollPositionTarget.